src/utils/helpers/scrollable-parent/_spec_.js

```diff
@@ -0,0 +1,23 @@
+// import Browser from '../browser';
```

Comment:
@adrianpastur What is to be done about this commented test? I covered this module in draggablecontext, but may also write module specific tests with your permission.